web/ui/src/lib/clustersMap/campus/angouleme.ts

```diff
@@ -0,0 +1,63 @@
+import { Campus } from '../campus';
+import { Cluster } from '../cluster';
+import { ICampus } from '../types';
+
+//
+export class Angouleme extends Campus implements ICampus {
+  emoji = (): string => '🇫🇷';
+
+  name = (): string => 'Angouleme';
+
+  extractorRegexp = (): RegExp =>
+    /(?<clusterWithLetter>c(?<cluster>\d+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>p(?<workspace>\d+))/i;
+
+  clusters(): Cluster[] {
+    return [
+      new Cluster({
+        identifier: 'c1',
+        name : "The silent one",
+        map: [
```

Comment:
Missing prettier ignore before map to prevent prettier to reformat the map
```suggestion
        // prettier-ignore
        map: [
```